src/BugsnagServiceProvider.php

```diff
@@ -372,6 +372,12 @@ protected function setupPaths(Client $client, $base, $path, $strip, $project)
      */
     protected function setupSessionTracking(Client $client, $endpoint, $events)
     {
+        // Session support removed in Lumen 5.3 - only setup automatic session
+        // tracking if the session function is avaiable
+        if (!function_exists('session')) {
+            return;
```

Comment:
I think it would be worth adding a log here to mention that sessions are not automatically being tracked, as someone may expect if they set the configuration option.

Reply:
I'm struggling to log anything (using Laravel logging) as it calls recursively into the Bugsnag setup code for the BugsnagLogger handling. Is there a lower-level approach that could be used to log?

Reply:
Also, given the fact that it doesn't currently work and there should be an alternative solution to allow support for this in the not-too-distant-future, I think it's not too important to log this if it proves difficult.

Reply:
Yeah, I hadn't considered the logging straight to Bugsnag aspect.  In theory it shouldn't be a problem if the log-level is high enough, but there's no guarantee there, and I haven't actually tried it myself.

In which case it's probably fine to not log.